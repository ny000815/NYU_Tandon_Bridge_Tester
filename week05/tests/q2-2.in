15
50
3
34
20
