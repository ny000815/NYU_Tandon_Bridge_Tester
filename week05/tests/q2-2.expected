I thought of a number between 1 and 100! Try to guess it.
Range: [1, 100], Number of guess left: 5
Your guess: Wrong! My number is bigger.

Range: [16, 100], Number of guess left: 4
Your guess: Wrong! My number is smaller.

Range: [16, 49], Number of guess left: 3
Your guess: Wrong! My number is bigger.

Range: [4, 49], Number of guess left: 2
Your guess: Wrong! My number is smaller.

Range: [4, 33], Number of guess left: 1
Your guess: Out of guesses! My number is 32
